### PR TITLE
beta-20161006 release notes

### PR DESCRIPTION
**Pull request by jseldess**
Based on https://github.com/cockroachdb/cockroach/issues/9752

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/728)

### CODE REVIEW

**Comment by tschottdorf** [comment]
^- That link does not work.

File: beta-20161006.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Although this week's release contains only a few internal changes and doc updates, note that we are preparing to merge numerous features from our `develop` branch to `master` for inclusion in an upcoming release.  
+
+### Binaries
+
+- [Mac](https://binaries.cockroachdb.com/cockroach-beta-20161006.darwin-10.9-amd64.tgz)
+- [Linux](https://binaries.cockroachdb.com/cockroach-beta-20161006.linux-amd64.tgz)
+
+### Internal Changes
+
+- Nodes now periodically exchange their configured maximum clock offset and exit fatally if they find another node operating with a different configuration. [#9612](https://github.com/cockroachdb/cockroach/pull/9612)
```
Comment (tamird) [inline beta-20161006.md]: s/exchange/compare/

**Comment by jseldess** [comment]
@tschottdorf, it works for me. Try again?

**Comment by andreimatei** [comment]
:lgtm: but see that deletion

---

Review status: 0 of 4 files reviewed at latest revision, 2 unresolved discussions.

---

*[_data/sidebar_doc.yml, line 353 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/728#-KTQa4ELyokSvjjiRR2B:-KTQa4EMxQXciApyh0hn:b-q4wq7g) ([raw file](https://github.com/cockroachdb/docs/blob/ebf2862ef10a6e2368bdaf974df07790ef39da9d/_data/sidebar_doc.yml#L353)):*
> ```YAML
>         - title: beta-20160908
>           url: /beta-20160908.html
> ```

why was this deleted?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/728#-:-KTQaG5qaJlpOyFMuDTO:b4nrr58)*

**Comment by jseldess** [comment]
*[_data/sidebar_doc.yml, line 353 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/728#-KTQa4ELyokSvjjiRR2B:-KTQaZ-_s-JMdznUGLhB:bxeuyde) ([raw file](https://github.com/cockroachdb/docs/blob/ebf2862ef10a6e2368bdaf974df07790ef39da9d/_data/sidebar_doc.yml#L353)):*
<details><summary><i>Previously, andreimatei (Andrei Matei) wrote…</i></summary>
> why was this deleted?
</details>
We always remove the oldest version from the sidebar and add it to `older-versions.md`. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/728)*

File: beta-20161006.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- Internal low-level instrumentation has been added for debugging performance bottlenecks. [#9638](https://github.com/cockroachdb/cockroach/pull/9638)
```
Comment (tamird) [inline beta-20161006.md]: these are not user-visible changes, i think we should omit them.

Comment (jseldess) [inline beta-20161006.md]: Without these, the notes would be nothing but doc updates. Plus, we've featured internal changes is a handful of previous releases.

File: beta-20161006.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: What's New in beta-20161006
+toc: false
+summary: Additions and changes in CockroachDB version beta-20161006.
+---
+
+## Oct 6, 2016
+
+Although this week's release contains only a few internal changes and doc updates, note that we are preparing to merge numerous features from our `develop` branch to `master` for inclusion in an upcoming release.
```
Comment (jseldess) [inline beta-20161006.md]: Will fix.

File: beta-20161006.md (hunk 1 above)
Comment (jseldess) [inline beta-20161006.md]: Will fix.

File: _data/strings.yml [hunk 4]
```diff
@@ -1,2 +1,2 @@
-version: beta-20160929
-build_time: 2016/09/29 17:54:45 (go1.7.1)
+version: beta-20161006
+build_time: 2016/10/06 116:33:49 (go1.7.1)
```
Comment (jseldess) [inline _data/strings.yml]: Doesn't your day have 116 hours? Fixing in separate PR. Pulled the trigger on this one already.

File: beta-20161006.md (hunk 2 above)
Comment (tamird) [inline beta-20161006.md]: The first bullet in the list is user-visible, but the last two are not.

**Pull request merged**